README.md

```diff
@@ -1,8 +1,33 @@
 # GitStats
-A simple PreactJS app that uses the GitHub search API to display repositories in a list.
 
-## Running
-`npm install` then `npm run start`
+## Introduction
+A simple PreactJS app that uses the GitHub search API to display repositories in a list. 
+
+![alt text](https://github.com/ivanseed/gitstats/blob/update-readme/src/assets/GitStatsView.png)
```

Comment:
One more minor thing please... Can you please set `alt text` to be `gitstats.io`.